pango-view/viewer-render.c

```diff
@@ -728,10 +756,14 @@ parse_options (int argc, char *argv[])
      "Enable single-paragraph mode",					NULL},
     {"text",		't', 0, G_OPTION_ARG_STRING,			&opt_text,
      "Text to display (instead of a file)",			    "string"},
+    {"text-file",   0, 0, G_OPTION_ARG_STRING,			&opt_text_file,
+     "Set input text file-name. If no text is provided, standard input is used for input.", "filename"},
```

Comment:
I think this option is also redundant since the first non-option argument is the text file to read from.

Reply:
Fixed